ui/js/services/taskcluster.js

```diff
@@ -47,8 +46,9 @@ const thTaskcluster = (() => {
       task.created = fromNow(0);
       task.deadline = fromNow('1 day');
 
-      _.map(task.payload.artifacts, function (artifact) {
+      task.payload.artifacts.map(function (artifact) {
```

Comment:
Since this is not using the return value, let's change it to a ``forEach`` with no return instead.  It's modifying the array in-place, so map really wasn't appropriate here in the first place.